response_operations_ui/controllers/collection_exercise_controllers.py

```diff
@@ -44,6 +44,58 @@ def download_report(collection_exercise_id, survey_id):
     return response
 
 
+def get_collection_exercise(short_name, period):
+    logger.debug(
+        "Retrieving collection exercise details", short_name=short_name, period=period
+    )
+    url = (
+        f'{app.config["BACKSTAGE_API_URL"]}/v1/collection-exercise/{short_name}/{period}'
+    )
+    response = requests.get(url)
+    if response.status_code != 200:
+        raise ApiError(response)
+
+    logger.debug(
+        "Successfully retrieved collection exercise details",
+        short_name=short_name,
+        period=period,
+    )
+    return response.json()
+
+
+def get_collection_exercise_events(short_name, period):
+    logger.debug('Retrieving collection exercise details for the event page',
+                 short_name=short_name, period=period)
+    url = f'{app.config["BACKSTAGE_API_URL"]}/v1/collection-exercise/{short_name}/{period}/events'
```

Comment:
Should be using the CE service directly now as on master: https://github.com/ONSdigital/response-operations-ui/blob/71f2d351aa6643f7211cd44efa3965b851c9203c/response_operations_ui/controllers/collection_exercise_controllers.py#L190

Reply:
Oh yeah I deleted this method and it keeps cropping back up.

Reply:
Sneaky :)